site/en/tutorials/keras/text_classification.ipynb

```diff
@@ -928,7 +928,7 @@
         "\n",
         "This tutorial showed how to train a binary classifier from scratch on the IMDB dataset. As an exercise, you can modify this notebook to train a multiclass classifier to predict the tag of a programming question on [Stack Overflow](http://stackoverflow.com/).\n",
```

Comment:
Suggestion: 

- If you can, write "multi-class" with a dash (on line 929, 927, and throughout the notebook) (source: https://www.tensorflow.org/s/results?q=multi-class - it's not 100% consistent on the site yet)